activity_calendar/templates/activity_calendar/activity_overview.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'core/base.html' %}
+{% load static %}
+{% load martor_render %}
+
+{% block title %}
+    Squire - Boardgames
+{% endblock title %}
+
+{% block content-frame-class %}
+    container-fluid
+{% endblock %}
+
+{% block content %}
+    <div class="m-auto" style="max-width: 1250px;">
+        <h1>Activities</h1>
+        <p>
+        Below is a list of the upcoming two weeks. For the calendar to all activities,
+            <a href="{% url 'activity_calendar:calendar' %}">
+                click here.
+            </a>
+        </p>
+
+        <div class="row">
+        {% for activity_moment in activities %}
+            <div class="col-12 col-md-6 col-xl-4 p-1">
+                {% include "activity_calendar/snippet_activity_moment.html" %}
+            </div>
+        {% endfor %}
+        </div>
+
+    </div>
+{% endblock %}
+
+
+
+{% block css %}
```

Comment:
I don't think we need Martor's CSS on this page; the activity descriptions aren't shown here.